src/main/scala/io/iohk/ethereum/transactions/PendingTransactionsManager.scala

```diff
@@ -2,26 +2,27 @@ package io.iohk.ethereum.transactions
 
 import akka.actor.{Actor, ActorRef, Cancellable, Props}
 import akka.util.{ByteString, Timeout}
+import com.google.common.cache.{Cache, CacheBuilder, RemovalCause, RemovalNotification}
```

Comment:
It seems we do not have this cache in `build.sbt` , so it seems is it some transitive dependency. Maybe it would be worth adding it, to not depend on transitive dependencies ?

Reply:
It is from `com.twitter.util-collection`. I could add it explicitly but in my opinion, it is unnecessary.